### DISCUSSION TITLE
Use of Milvus logo

### DISCUSSION THREAD

**Discussion by timheuer** [milvus-io/milvus#33095] [Q&A and General discussion]
Hello, I'm curious if I could get a contact regarding using the Milvus logo in a package that leverages Milvus/Milvus C# client.  I'm specifically looking to actually use the square logo used here on GitHub for the milvus-io org.  Looking for usage terms and attribution requirements.

**Comment by xiaofan-luan** [comment]
Hi Tim.
I believe the c# client is under milvus-io and for sure you can use te milvus logo.
Let me know if you need any extra help on this. I'm the maintainer of Milvus project

**Reply by timheuer** [reply]
Thank you @xiaofan-luan! -- Just to clarify this will be a new Package that does some lightweight config/healthchecks reporting, etc. for our framework.  Is there any credit you'd prefer to exist in the package that says something like "Milvus and the Milvus logo are copyright and used with permission from the Milvus project." or something specific?

**Reply by xiaofan-luan** [reply]
Right now all right preseved by LF AI & Data foundation.  And I don't see any strict regulation for now. Thanks for asking in advance
